test/sql/blob/test_blob.cpp

```diff
@@ -0,0 +1,268 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+#include "duckdb/main/prepared_statement.hpp"
+
+#include <fstream>
+
+using namespace duckdb;
+using namespace std;
+
```

Comment:
Please add some tests with persistent storage of blobs (see `test_storage.cpp`).